test/clj/rems/application/test_cleaner.clj

```diff
@@ -0,0 +1,80 @@
+(ns ^:integration rems.application.test-cleaner
+  (:require [clj-time.core :as time]
+            [clojure.test :refer :all]
+            [rems.db.core :as db]
+            [rems.db.events :as events]
+            [rems.db.applications :as applications]
+            [rems.application.cleaner :as cleaner]
+            [rems.db.testing :refer [test-db-fixture rollback-db-fixture]]))
+
+(use-fixtures :once test-db-fixture)
+(use-fixtures :each rollback-db-fixture)
+
+(defn- add-dummy-draft-application!
+  [{:keys [date-time actor]}]
+  (let [app-id (:id (db/create-application!))
+        created-event {:event/type :application.event/created
+                       :event/time date-time
+                       :event/actor actor
+                       :application/id app-id
+                       :application/external-id ""
+                       :application/resources []
+                       :application/licenses []
+                       :application/forms []
+                       :workflow/id 0
+                       :workflow/type :workflow/default}
+        draft-saved-event {:event/type :application.event/draft-saved
+                           :event/time date-time
+                           :event/actor actor
+                           :application/id app-id
+                           :application/field-values []}]
+    (events/add-event! created-event)
+    (events/add-event! draft-saved-event)
+    app-id))
+
+(defn- add-dummy-application!
+  [{:keys [date-time actor]}]
+  (let [app-id (:id (db/create-application!))
+        created-event {:event/type :application.event/created
+                       :event/time date-time
+                       :event/actor actor
+                       :application/id app-id
+                       :application/external-id ""
+                       :application/resources []
+                       :application/licenses []
+                       :application/forms []
+                       :workflow/id 0
+                       :workflow/type :workflow/default}
+        draft-saved-event {:event/type :application.event/draft-saved
+                           :event/time date-time
+                           :event/actor actor
+                           :application/id app-id
+                           :application/field-values []}
+        submitted-event {:event/type :application.event/submitted
+                         :event/time date-time
+                         :event/actor actor
+                         :application/id app-id}]
+    (events/add-event! created-event)
+    (events/add-event! draft-saved-event)
+    (events/add-event! submitted-event)
+    app-id))
+
+(deftest cleaner
```

Comment:
We use the `test-` prefix usually (as a convention to make it easy to search for tests).